tests/lib/rules/no-loss-of-precision.js

```diff
@@ -0,0 +1,150 @@
+/**
+ *@fileoverview Tests for no-loss-of-precision rule.
+ *@author Jacob Moore
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-loss-of-precision"),
+    { RuleTester } = require("../../../lib/rule-tester");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-loss-of-precision", rule, {
+    valid: [
+        "var x = 12345",
```

Comment:
Would it make sense to have some non-number assignments to be sure this rule doesn't false trigger?

Reply:
Agreed that regression tests would be nice to have.

Reply:
Added!